src/Pages/Member/Login.js

```diff
@@ -1,9 +1,168 @@
+      <div className="Login">
+        <img
+          className="login-logo"
+          src="https://wiselyshave-cdn.s3.amazonaws.com/assets/images/WiselyLogo.svg"
+        />
+        <div className="login-container">
+          <div className="login-contents">
+            <p>
+              <strong className="font-bold font-size-46">
+                로그인 및 회원가입
+              </strong>
+              <span className="font-bold-light font-size-46">을</span>
+              <div className="font-bold-light font-size-46">시작합니다</div>
+            </p>
+            <div className="kaka-start-btn start-btn">
+              <img
+                className="login-icon"
+                src="https://wiselyshave-cdn.s3.amazonaws.com/assets/images/signIn/signInKakao.png"
+              />
+              <div className="kakao-start-btn-text font-bold-light">
+                카카오로 <span className="font-bold">간편시작</span>
+              </div>
+              <div className="icon-right" />
+            </div>
+            <div
+              className="email-start-btn start-btn"
+              onClick={this.displayhandler}
+            >
+              <img
+                className="login-icon"
+                src="https://wiselyshave-cdn.s3.amazonaws.com/assets/images/signIn/signInEmail.png"
+              />
+              <div className="email-start-btn-text font-bold-light">
+                이메일로 시작하기
+              </div>
+              <img
+                className={
+                  this.state.displayActive ? "icon-right" : "icon-transform"
+                }
+                src="https://wiselyshave-cdn.s3.amazonaws.com/assets/images/arrow/arrowDownPurple.svg"
+              />
+            </div>
+          </div>
+          <div
+            className={
+              this.state.displayActive ? "email-box-visible" : "email-box-none"
+            }
+          >
+            <div
+              className={
+                userid.includes("@") &&
+                userid.includes(".co") &&
+                userid.length >= 10
+                  ? "email-input-container-blue"
+                  : "email-input-container"
+              }
+            >
+              <input
+                type="text"
+                placeholder="이메일"
+                className="email-input"
+                name="userid"
+                onChange={this.inputHandler}
+              />
+              <img
+                className={
+                  userid.includes("@") &&
+                  userid.includes(".co") &&
+                  userid.length >= 10
+                    ? "check-img-visible"
+                    : "check-img-none"
+                }
+                src="https://wiselyshave-cdn.s3.amazonaws.com/assets/images/checkBlue.svg"
+              />
+            </div>
+            <div
+              className={
+                password.length >= 6
+                  ? "pw-input-container-blue"
+                  : "pw-input-container"
```

Comment:
```suggestion
                  ? "pw-input-container blue"
                  : "pw-input-container"
```
이와 같이 작성하면 `pw-input-container`에 공통되는 스타일을 지정해두고 blue 클래스가 붙을 때 변화하는 요소만 따로 지정해줄 수 있습니다. nesting 활용해서 고쳐주세요. 이와 유사한 케이스도 전부 고쳐주세요.